lib/tasks/osc.rake

```diff
@@ -238,4 +238,14 @@ namespace :osc do
       end
     end
   end
+
+  desc "Print the current packaging configuration"
+  task :config do
+    puts "Package directory: #{package_dir}"
+    puts "OBS instance:      #{obs_api}"
+    puts "OBS project:       #{obs_project}"
+    puts "OBS package name:  #{package_name}"
+    puts "OBS build target:  #{build_dist}"
+    puts "OBS submit target: #{obs_sr_project}"
+  end
```

Comment:
idea looks fine for me. Implementation not so much. I prefer if it is possible to use directly all config from https://github.com/openSUSE/packaging_rake_tasks/blob/master/lib/packaging/configuration.rb so I would add there method like "inspect" which can be printed here.

Reply:
Hm, I consider the `inspect` method primarily for debugging, e.g. it might include some internal data or implementation details, like `"#<Bar:0x0300c868 @bar=1>"`.

This is something displayed to the end user, I'd prefer to print only the relevant data and in a nice formatted way.

Reply:
still I think that config class itself suits better for this (maybe just not inspect method). As options are not osc specific only.